pkg/deploy/generate.go

```diff
@@ -0,0 +1,9 @@
+//go:generate go run ../../hack/gendeploy ../../deploy
+//go:generate go get github.com/go-bindata/go-bindata/go-bindata
+//go:generate go-bindata -nometadata -pkg $GOPACKAGE -prefix data ../../deploy
+//go:generate gofmt -s -l -w bindata.go
```

Comment:
1- leave these lines in /deploy, then you don't need all the outputDir changes - please revert these.
2- don't do go get.  Vendor go-bindata and call the vendored version.  See `required` stanza in Gopkg.toml.

Reply:
The move was intentional but I can play with go-bindata falgs to achieve what you are asking for.